### PR TITLE
add fuse pause

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

File: go/citogo/main.go [hunk 1]
```diff
@@ -277,6 +279,10 @@ func (r *runner) runTests() error {
 		if err != nil {
 			return err
 		}
+		if r.opts.Pause > 0 {
+			fmt.Printf("PAUS: %ds\n", r.opts.Pause)
```
Comment (strib) [inline go/citogo/main.go]: PAUS -> PAUSE

File: Jenkinsfile [hunk 2]
```diff
@@ -607,6 +607,7 @@ def testGo(prefix, packagesToTest) {
         'github.com/keybase/client/go/kbfs/libfuse': [
           flags: '',
           timeout: '10m',
+          citogo_extra : '--pause 2s'
```
Comment (strib) [inline Jenkinsfile]: This only needs to be .25s I think, at least on my local machine.  But I guess this should be fine too, will just make CI take longer (but maybe is better at avoiding flakes?).

File: go/citogo/main.go [hunk 3]
```diff
@@ -67,6 +68,7 @@ func (r *runner) parseArgs() (err error) {
 	flag.BoolVar(&r.opts.NoCompile, "no-compile", false, "specify flag if you've pre-compiled the test")
 	flag.StringVar(&r.opts.TestBinary, "test-binary", "", "specify the test binary to run")
 	flag.StringVar(&r.opts.Timeout, "timeout", "60s", "timeout (in seconds) for any one individual test")
+	flag.IntVar(&r.opts.Pause, "pause", 0, "pause that many seconds between each test")
```
Comment (strib) [inline go/citogo/main.go]: Does an `IntVar` work with "2s" passed in?  Maybe it should be a `DurationVar`?

File: go/kbfs/libfuse/mount_test.go [hunk 4]
```diff
@@ -4136,4 +4136,5 @@ func TestUpdateHistoryFile(t *testing.T) {
 	err = json.Unmarshal(b, &histAll)
 	require.NoError(t, err)
 	require.Len(t, histAll.Updates, 11)
+	t.Logf("hi")
```
Comment (strib) [inline go/kbfs/libfuse/mount_test.go]: XXX

File: go/citogo/main.go (hunk 1 above)
Comment (maxtaco) [reply go/citogo/main.go]: 4 char width on all of these!

File: Jenkinsfile (hunk 2 above)
Comment (maxtaco) [reply Jenkinsfile]: i was concerned that CI is crappier than your machine.

File: go/citogo/main.go (hunk 3 above)
Comment (maxtaco) [reply go/citogo/main.go]: mess up, thx

File: go/citogo/main.go (hunk 1 above)
Comment (strib) [reply go/citogo/main.go]: Ah sorry!

File: go/kbfs/libfuse/mount_test.go (hunk 4 above)
Comment (maxtaco) [reply go/kbfs/libfuse/mount_test.go]: there to trigger a CI run on fuse

**Pull request merged**